### ISSUE TITLE
failed to pull the signing certificate to determine your team ID: exit status 44

### ISSUE THREAD

**Issue by baolitoumu** [fyne-io/fyne#900]
### Describe the bug:

failed to pull the signing certificate to determine your team ID: exit status 44

My xcode hasn't been developed, it's just a simple setup, does it need to be completely configured?

### Screenshots:

![image](https://user-images.githubusercontent.com/14047916/80190382-d05fe680-8646-11ea-8170-3b1a9325daa0.png)

### Example code:
```go
package main

import (
	"fyne.io/fyne"
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)

func main() {
	a := app.New()

	w := a.NewWindow("Hello")
	w.SetContent(&widget.Box{Children: []fyne.CanvasObject{
		&widget.Label{Text: "Hello Fyne!"},
		&widget.Button{Text: "Quit", OnTapped: func() {
			a.Quit()
		}},
	}})

	w.ShowAndRun()
}
```

### Device 
 - MAC OS: [10.15.4 ]
 - Go version: [e.g. 1.14.2]
 - Fyne version: [e.g. 1.2.0 or git SHA]

**Comment by andydotxyz** [comment]
To package your app for installation you need an XCode certificate installed.
If you haven't already you would need to get a certificate and provisioning profile for the appID (or a wildcard). Once you have created them they need to be installed on your computer (just download them and double click).
This whole process needs to be improved for sure! Some more details are in our README-mobile file.

**Comment by andydotxyz** [comment]
If by any chance you have one of the newer "Apple Development" certificates then apologies - a bug was found and getting fixed for 1.3.

**Comment by andydotxyz** [comment]
Probably fixed (if it was a bug) by f1b1e909, which is on develop for the 1.3 release
